Jenkinsfile

```diff
@@ -220,31 +223,51 @@ node {
           Utils.markStageSkippedForConditional("memcheckReleaseTest")
         }
       }
+    } finally {
+      sh "ls -A1 | xargs rm -rf"
+      deleteDir()
+    }
+  }
+}
 
-      stage("Cleanup") {
-        // Clean up workspace.
-        script {
-          githubNotify context: 'CI Pipeline', status: 'SUCCESS'
-          if (env.BRANCH_NAME == 'master' || full_ci) {
-            githubNotify context: 'Full CI', status: 'SUCCESS'
-          }
-        }
-        step([$class: 'WsCleanup'])
+// I have not found a nice way to run this in parallel with the steps above, as it will those are in a docker.inside block and this is not.
+node('mac') {
+  stage("clangDebugMac") {
+    if (env.BRANCH_NAME == 'master' || full_ci) {
+      try {
+        checkout scm
+
+        // We do not use install.sh here as there is no way to run OS X in a Docker container
+        sh "git submodule update --init --recursive --jobs 4"
+
+        sh "mkdir clang-debug && cd clang-debug && /usr/local/bin/cmake -DCMAKE_CXX_COMPILER_LAUNCHER=/usr/local/bin/ccache -DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_COMPILER=/usr/local/Cellar/llvm\\@7/7.0.1/bin/clang -DCMAKE_CXX_COMPILER=/usr/local/Cellar/llvm\\@7/7.0.1/bin/clang++ .."
+        sh "cd clang-debug && CCACHE_CPP2=yes CCACHE_SLOPPINESS=file_macro PATH=/usr/local/bin:$PATH make -j libjemalloc-build"
+        sh "cd clang-debug && CCACHE_CPP2=yes CCACHE_SLOPPINESS=file_macro make -j4"
+        sh "cd clang-debug && ./hyriseTest"
```

Comment:
Any specific reason for not using the typical `./clang-debug/hyriseTest`?

Reply:
None in particular. Will change.